apps/mochi-web/components/explore/index/sections/TransactionSection.tsx

```diff
@@ -61,6 +61,7 @@ export const TransactionSection = () => {
           totalItems: total,
           onItemPerPageChange: setSize,
           onPageChange: setPage,
+          allowCustomPage: true,
```

Comment:
The mobile navigation should be center

<img width="552" alt="image" src="https://github.com/consolelabs/mochi-ui/assets/12707960/d2bdf74e-ad2e-4c5d-9af6-365e637ae315">